src/Breadcrumbs.php

```diff
@@ -127,7 +129,7 @@ class Breadcrumbs extends Widget
      */
     public function run(): string
     {
-        if (empty($this->links)) {
+        if (!$this->links) {
```

Comment:
Please don't compare any types as a boolean. If you want to check empty array or not, use `$var === []` or `empty($var)` or `count($var) === 0`. 
`!$var` too implicit expression IMHO.

Reply:
Done.